src/installer/tests/TestUtils/RuntimeConfig.cs

```diff
@@ -279,9 +279,7 @@ public void Save()
 
             if (_tfm is not null)
             {
-                runtimeOptions.Add(
-                    Constants.Tfm.RuntimeConfigPropertyName,
-                    _tfm);
```

Comment:
Are you sure that the runtime option isn't used in this test?

Reply:
it was replaced with simple `"tfm"` on the line below

Reply:
Right, sorry didn't see that.